src/library/Uno.Material/Controls/BottomNavigationBarItem.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls.Primitives;
+
+namespace Uno.Material.Controls
+{
+	public partial class BottomNavigationBarItem : ToggleButton
+	{
+		public string Data
```

Comment:
Let's say someone would like to bind using the index, they could bind to the SelectedItem, check the "name" property instead of the index no? (assuming we give a name to the BottomNavigationBarItem)